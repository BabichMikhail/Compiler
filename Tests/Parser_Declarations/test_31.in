const
	a : array[1..3] of integer = (0, 1, 2);
	b = a;
	c = 2;
var 
	e: integer;
const 
	d : array[10..12] of integer = (0, 1, c);
var 
	arr : array [100..100*c] of array [1..2] of array [ 10..20] of array[ -10..10] of char;