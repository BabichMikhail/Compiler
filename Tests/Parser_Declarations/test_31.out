Const	a
0
1
2

Const	b
a

Const	c
2

Var	e
Type	integer

Const	d
0
1
c

Var	arr
Array	
100
   c
*
   100
Array	
1
2
Array	
10
20
Array	
   10
-
10
Type	char
a

